nototools/generate_website_2_data.py

```diff
@@ -994,10 +996,17 @@ def build_family_images(
         lspc = 32
       image_file_name = '%s_%s_%d_%s.%s' % (
           family_id, lang_scr, weight, style, imgtype)
-      if is_cjk:
-        family_suffix = ' ' + font.weight
+      if is_cjk and family.name.find('Serif') < 0:
+        # The sans and serif cjk's are named differently, and it confuses
+        # fontconfig.  Sans includes 'Regular' and 'Bold' in the standard
+        # font names, but serif doesn't.  Fontconfig registers two names
+        # for these in the sans (with and without the weight), but only
+        # the name without weight for the serif.  So if you ask pango/cairo
+        # for the serif font and include the weight name, it fails and
+        # falls back to some non-noto font.
+        family_name = family.name + ' ' + font.weight
```

Comment:
In an ideal situation, all the weights should be style-locked, but the 'history' of name ids makes it harder as you know. Ken wants to move intentionally slowly on this.